docs/reference/api/pages.md

```diff
@@ -220,7 +220,7 @@ Individual page objects will include `_children` and `_ancestor` arrays, as well
 
 ```javascript
 // Request inside an async function.
-const response = await fetch('http://example.net/api/v1/@apostrophecms/page/ckitdo5oq004pu69kr6oxo6fr:en:published?apikey=myapikey', {
```

Comment:
Wait, you lost me here. If you have a complete `_id` you may send it and that information is sufficient to ensure you get the right locale and mode. The query parameters are for situations where there's no `_id` or you want to override some part of what it implies.

Reply:
Corrected back to the original.